_includes/url/url-ai2.html

```diff
@@ -0,0 +1 @@
+href="http://allenai.org/" target="_blank"
```

Comment:
What is this file?

Reply:
Abstracted URL so I could change it once.

Reply:
Will take a stab at using Jekyll variables instead in next pass.